run-tests.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+IOT_SCRIPT_PATH="src/iot-setup.py"
+GGL_SCRIPT_PATH="src/ggl-setup.py"
+JSON_FILE="iot_setup_data.json"
+
+# Get all test cases
+# TODO: get all tests from all test files
+get_test_functions() {
+    grep -o "test_[[:alnum:]_]*" ./src/aws-greengrass-testing-security.py | sort -u
+}
+
+setup_and_cleanup() {
+    local test_name=$1
+
+    echo "Running IoT setup..."
+    python3 $IOT_SCRIPT_PATH set_up_core_device --region=$AWS_REGION
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Double quote to prevent globbing and word splitting.